packages/labs/observers/README.md

```diff
@@ -0,0 +1,49 @@
+# @lit-labs/observers
+
+A set of reactive controllers that facilitate using the platform observer
+objects, including:
+
+- MutationObserver
+- ResizeObserver
+- IntersectionObserver
+- PerformanceObserver
+
+## Overview
+
+The modern web platform provides a number of observer helpers that can be used
+to detect changes to which web applications may want to react. By managing
+one of these observers with a reactive controller, changes can be easily
+integrated into the Lit reactive lifecycle. The controller can also help
+manage observer cleanup and rendering in response to changes.
+
+## Installation
+
+From inside your project folder, run:
+
+```bash
+$ npm install @lit-labs/observers
+```
+
+## Usage
+
+Here's an example:
+
+```ts
+import {MutationController} from '@lit-labs/observers';
+// ...
+
+class MyElement extends LitElement {
+  private _observer = new MutationController(this, {
+    target: this,
```

Comment:
since `this` is the default, don't include it?

Reply:
It's not the default. If you don't specify it then it's just not set and you can manually call `observe`.

Reply:
Interesting. I think `this` would be a reasonable default.

Reply:
And `null` if you want to manually observe something else?

Reply:
I think that makes sense. I'm just guessing here that the most common thing people will want to observe is the host though. Especially for IntersectionObserver.

Reply:
Discussed offline: resolved to make `this` the default and explicit `null` not observe anything by default.

Reply:
Done.

Reply:
Still says `target: this` here